### PR TITLE
[FIX] #1690 - Also check pdo_pgsql extension in check_install()

### PR DESCRIPTION
**Pull request by blankoworld**
After this pull request: https://github.com/FreshRSS/FreshRSS/pull/1691

Create a new one with dev branch and without previous PR proposals.

Hope all is OK now!

### CODE REVIEW

File: lib/lib_rss.php [hunk 1]
```diff
@@ -398,11 +398,12 @@ function is_referer_from_same_domain() {
 function check_install_php() {
 	$pdo_mysql = extension_loaded('pdo_mysql');
 	$pdo_sqlite = extension_loaded('pdo_sqlite');
+	$pdo_pgsql = extension_loaded('pdo_pgsql');
```
Comment (Frenzie) [inline lib/lib_rss.php]: Is this order (also below) the most sensible? With only three I suppose it doesn't matter but I'd go for alphabetic. :-)

**Comment by Frenzie** [comment]
You can force push onto a branch btw. Something like this.

```bash
git checkout messed-up-branch
git checkout -b backup-branch # just in case
git checkout messed-up-branch
git reset HEAD~5 # or a few more, whatever
git pull upstream dev # you'll need to set up a remote called upstream for that to work of course
git cherry-pick e5ba61e # apply the one commit on top
git push -f
```

Alternatively you can use `git rebase -i HEAD~5`; depends a bit on what you want.

**Comment by blankoworld** [comment]
Wow, I see. A little bit complex for me, but hey, I found a way to give you the change you expect. I think. Probably :).

**Comment by Alkarex** [comment]
Many thanks, @blankoworld 👍 
(If you want), please add your name to the contributor list: https://github.com/FreshRSS/FreshRSS/blob/master/CREDITS.md

**Comment by Alkarex** [comment]
https://github.com/FreshRSS/FreshRSS/issues/1690

**Review by aledeg — APPROVED** [review]
Looks good to me. Thank you for your effort.

File: lib/lib_rss.php [hunk 2]
```diff
@@ -397,12 +397,13 @@ function is_referer_from_same_domain() {
...
 function check_install_php() {
 	$pdo_mysql = extension_loaded('pdo_mysql');
+	$pdo_pgsql = extension_loaded('pdo_pgsql');
 	$pdo_sqlite = extension_loaded('pdo_sqlite');
 	return array(
 		'php' => version_compare(PHP_VERSION, '5.3.8') >= 0,
 		'minz' => file_exists(LIB_PATH . '/Minz'),
 		'curl' => extension_loaded('curl'),
-		'pdo' => $pdo_mysql || $pdo_sqlite,
+		'pdo' => $pdo_mysql || $pdo_sqlite || $pdo_pgsql,
```
Comment (Frenzie) [inline lib/lib_rss.php:406]: This one's still at the end. :-P

**Review by Frenzie — APPROVED** [review]
Looks good. Don't mind me going a bit overboard. xD

File: lib/lib_rss.php (hunk 2 above)
Comment (Alkarex) [reply lib/lib_rss.php:406]: In the rest of the code, there are many places with that order, for historical reasons (MySQL was used first, and then SQLite compatibilité, and last PostgreSQL). So I do not mind either way :-)

Comment (Frenzie) [reply lib/lib_rss.php:406]: It's one of those things where on the one hand it seems like an enormous nitpick. On the other hand I've learned through bitter experience how innocent code where different people just randomly add some stuff at the end or bottom can become an unparseable blob before you good and well realize what happened.

You could quite validly point out that the set will always remain minuscule, but it still serves as an example that might be followed.

Anyway, that's just a braindump. I already explicitly marked it approved after all. ;-)

**Comment by blankoworld** [comment]
> (If you want), please add your name to the contributor list: https://github.com/FreshRSS/FreshRSS/blob/master/CREDITS.md

How should I add it? A new commit above this pull request? Or into the same commit with ```git commit --amend```? What's the best?

**Comment by Frenzie** [comment]
Either way is fine. I prefer more smaller commits but that's just me. ;-)

**Comment by blankoworld** [comment]
Seems that all is fine for me. Hope everything is OK for you now ;)

**Comment by Frenzie** [comment]
Thanks!

**Pull request merged**